domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Responses/TransferAgreementAuditsResponse.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace API.ApiModels.Responses;
+
+public record TransferAgreementAuditsResponse(List<TransferAgreementAuditDto> Result);
```

Comment:
I think this response should be changed to having 

- a property for the ID of transfer agreement
- a property called `Changes` or `History` or `Audits` that return a `List<TransferAgreementAuditDto>`

Reply:
As per our conversation, we agreed upon not including the transferagreementId, in result response itself, as it is contained within its list. will Resolve this conversation.